Add 22 Domain of CV DOIT PERSADA INDONESIA

doitindonesia opened the pull request:
Public Suffix List (PSL) Pull Request (PR) Template
====

Each PSL PR needs to have a description, rationale, indication of DNS validation and syntax checking, as well as a number of acknowledgements from the submitter.  This template must be included with each PR, and the submitting party MUST provide responses to all of the elements in order to be considered.

<!-- #### READ THIS FIRST ####

If you haven't yet, please read our guidelines:
https://github.com/publicsuffix/list/wiki/Guidelines#submit-the-change

Also, read them again, as many skip that part and 
get confused about why their PR is delayed or does
not get accepted when theirs didn't follow them.

A recent PR using the current template is 
https://github.com/publicsuffix/list/pull/1591, although 
the organization and description were not as substantial 
as desired, which required maintainers time to visit the 
requestors website to further research. 
Having more robust org/desc improves the PR processing 
pace due to the extra cycles not lost to research.
For an example of what an excellent description in a PR looks like
see https://github.com/publicsuffix/list/pull/615, 
although that example uses an earlier template.
-->
### Checklist of required steps

* [x] Description of Organization
* [x] Robust Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)

* [x] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration, and we shall keep the \_PSL txt record in place in the respective zone(s) in the affected section

__Submitter affirms the following:__ 
<!--
Third-party Limits are used elsewhere, such as at Cloudflare, Let's 
Encrypt, Apple, GitLab or others, and having an entry in the PSL alters 
the manner in which those third-party systems or products treat 
a given domain name or sub-domains within it.

To be clear, it is appropriate to address how those limits impact 
your domain(s) directly with that third-party, and it is inappropriate 
to submit entries to the PSL as a means to work around those limits or 
restrictions.
-->
  * [x] We are listing *any* third-party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)

<!--
The purpose of the question above is to expose limit workarounds.
If there are third party limits that the PR seeks to overcome, those
must be listed within the rationale section of this request, and 
provide a good level of detail the effort that was made to work directly 
with the third part(y|ies) in attempting to address this within their 
rationale response below.
In all cases, software and services should be discouraged from use of
the PSL as a rate-limiting tool, and provide clear instructions to their
own clients, partners and users on the manner in which they can directly
request rate limit increases.
We treat the following as an attestation in the public record of the 
requesting party that they are not attempting to bypass rate limits through
the PR.
-->

  * [x] This request was _not_ submitted with the objective of working around other third-party limits

<!--
The guidelines describe which section to place the entry, what the 
order of commented org placement, order of sorting of entries. 
(hint: TLD then SLD, Ascending sort)   Although it seems pedantic, 
the sorting and formatting rules help ensure all of the automation 
that uses the PSL operates correctly.  Typically both are solved or
neither.
-->

  * [x] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms
  * [x] The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting and sorting

<!-- 
Sorting and formatting of the entries is outlined in the guidelines 
and non-conforming requests are one of the largest sources of delay,
so getting this right initially will aid successfully having it 
proceed.  Miss-located entries and trailing spaces should be avoided.
-->

---

For Private section requests that are submitting entries for domains that match their organization website's primary domain, please understand that this can have impacts that may not match the desired outcome and take a long time to rollback, if at all.

To ensure that requested changes are entirely intentional, make sure that you read the affectation and propagation expectations, that you understand them, and confirm this understanding. 

PR Rollbacks have lower priority, and the volunteers are unable to control when or if browsers or other parties using the PSL will refresh or update.

<!-- 
Seriously, carefully read the downline flow of the PSL and the 
guidelines. Your request could very likely alter the cookie and 
certificate (as well as other) behaviours on your core domain name in 
ways that could be problematic for your business.

Rollback is really not predictable, as those who use or incorporate 
the PSL do what they do, and when. It is not within the PSL volunteers' 
control to do anything about that.  

The volunteers are busy with new requests, and rollbacks are lowest 
priority, so if something gets broken by your PR, it will potentially 
stay that way for an indefinite period of time (typically long).
-->

(Link: [about propagation/expectations](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion))

 * [x] *Yes, I understand*.  I could break my organization's website cookies etc. and the rollback timing, etc is acceptable.  *Proceed*.
---


<!--

As you complete each item in the checklist please mark it with an X

Example:

* [x] Description of Organization

-->

Description of Organization
====
doIT Persada Indonesia is a company engaged in the field of IT services for cloud hosting providers, website creation and management, digital marketing solutions.


Organization Website: 
====
Provide the website address of 
https://doit.co.id


Reason for PSL Inclusion
====
As our cloud hosting service, website and managed service, and digital marketing service, customers can either a) use their own domains, or b) choose one of the domains we own, and issue a subdomain to obtain the dedicated domain without managing their own domain. In the latter case, each subdomain on the shared domain belongs to a different customer. It is necessary for our domains to be added to the list, in order to prevent multi-site cookies.

DNS Verification via dig
=======

```
dig +short TXT _psl.azkasyah.co.id
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.ciruasland.co.id
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.doit.co.id
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.dopay.co.id
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.margalingkarjakarta.co.id
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.mikatama.co.id
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.petrolin.co.id
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.rehlata.co.id
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.specta.co.id
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.tmacontractor.co.id
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.mixstore.id
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.nobar.web.id
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.domail.biz.id
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.acisindonesia.com
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.akrilikdisplay.com
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.akrilikexpress.com
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.auliadigitalprint.com
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.duniaacrylic.com
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.dutaled.com
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.icebergwindowfilms.com
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.paketsolusi.com
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.royalbravosolusindo.com
https://github.com/publicsuffix/list/pull/1707
```
-->

Results of Syntax Checker (`make test`)
=========
```
cd linter;                                \
  ./pslint_selftest.sh;                     \
  ./pslint.py ../public_suffix_list.dat;
test_NFKC: OK
test_allowedchars: OK
test_dots: OK
test_duplicate: OK
test_exception: OK
test_punycode: OK
test_section1: OK
test_section2: OK
test_section3: OK
test_section4: OK
test_spaces: OK
test_wildcard: OK
test -d libpsl || git clone --depth=1 https://github.com/rockdaboot/libpsl;   \
  cd libpsl;                                                                    \
  git pull;                                                                     \
  echo "EXTRA_DIST =" >  gtk-doc.make;                                          \
  echo "CLEANFILES =" >> gtk-doc.make;                                          \
  autoreconf --install --force --symlink;
Already up to date.
Can't exec "autopoint": No such file or directory at /usr/share/autoconf/Autom4te/FileUtils.pm line 293.
autoreconf: error: autopoint failed with exit status: 2
make: *** [Makefile:18: libpsl-config] Error 2
```

doitindonesia committed:
Add doit.co.id

doitindonesia committed:
Merge branch 'publicsuffix:master' into master

doitcoid commented:
Hi, We just edited sorting our domain, please check.

doitindonesia committed:
Fix Failed Sorting

doitcoid commented:
Hi, We just edited sorting, please check. Thank you.

doitindonesia reviewed (commented):
We just edited Sorting. Thank you

dnsguru reviewed (changes requested):
NO, THIS DOES NOT GO IN ICANN SECTION

MOVE THESE TO PRIVATE SECTION, Sorted by the organization name, then these need to be sorted by TLD, then SLD, then 3LD within the Pull Request.  

PLEASE READ GUIDELINES to see if there is more you need to do in making a conforming Pull Request.   

Once this is done, use patience, as these wasted cycles and back and forth as well as the very long queue of Pull Requests ahead of this request - all work against the time available from volunteer maintainers.

dnsguru commented on public_suffix_list.dat:
```diff
@@ -1,4 +1,4 @@
-// This Source Code Form is subject to the terms of the Mozilla Public
+2// This Source Code Form is subject to the terms of the Mozilla Public
```
This would get immediately rejected, remove this 2

doitcoid replied:
I just edited, thank you

dnsguru commented on public_suffix_list.dat:
```diff
@@ -867,6 +867,31 @@ int.cv
 nome.cv
 org.cv
 
+// CV DOIT PERSADA INDONESIA : https://doit.co.id
```
Andhy please move the whole submitted section out of the ICANN section and down into the #PRIVATE section ordered by where your organization's name would be the next alphabetical org based upon other entries' commented organization name.

doitcoid replied:
I just edited too, thank you

doitindonesia committed:
Update public_suffix_list.dat

This would get immediately rejected, remove this 2 (Done)
#PRIVATE section ordered (Done)

doitindonesia reviewed (commented):
We just edited, thank you.

dnsguru commented:
Almost....  and these iterations are not something we are resourced for.

you moved section but need to alphabetically order the sorting of the
entries
.com
.biz.id
.co.id
Etc

|Entries Sorted by the organization name, 《DONE》
|then these need to be sorted by TLD,
|then SLD,
|then 3LD within the Pull Request.

On Fri, Feb 24, 2023, 9:49 PM doIT Nusantara ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In public_suffix_list.dat
> <https://github.com/publicsuffix/list/pull/1707#discussion_r1117871750>:
>
> > @@ -867,6 +867,31 @@ int.cv
>  nome.cv
>  org.cv
>
> +// CV DOIT PERSADA INDONESIA : https://doit.co.id
>
> I just edited too, thank you
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/publicsuffix/list/pull/1707#discussion_r1117871750>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AACQTJNGB3Y7PPUEYMOJJSDWZGMQJANCNFSM6AAAAAAVEIC2MU>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

doitindonesia committed:
Update public_suffix_list.dat

Alphabetically order the sorting of the entries (Done)
Thank you

doitindonesia reviewed (commented):
alphabetically order the sorting of the entries (Done)

kenjiew commented:
Sekedar koreksi @doitindonesia 
sorry @dnsguru , I use Indonesian

Meskipun ini agak telat, tapi saya cuma mau menjelaskan dengan bahasa yang agak simple, sebenarnya @dnsguru ingin menjelaskan ke @doitindonesia untuk mengurutkan dari alfabet a-z (setahu saya) dan mengurutkan urutan dari yang paling pendek hingga panjang (misal yang pendek ada contoh.com yang panjang itu contoh.co.id dan lain sebagainya) jadi urutan yang benar adalah

a.com
b.org
c.id
d.co.id
e.co.id

jadi intinya harus urut antara a-z, 2LD dan 3LD

meskipun ini sudah usang, tapi saya sedang ada waktu luang, hehe, jadi saya menyempatkan untuk membalas ini
